.gitignore

```diff
@@ -1,3 +1,5 @@
+.DS_Store
+examples/*/.env
```

Comment:
For the examples I've been adding a `.gitignore` in the example directory to keep them self contained.